smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolUnitTestResponseGenerator.kt

```diff
@@ -119,82 +109,12 @@ open class HttpProtocolUnitTestResponseGenerator protected constructor(builder:
     }
 
     protected open fun renderAssertions(test: HttpResponseTestCase, outputShape: Shape) {
-        val nestedShapes = model.getNestedShapes(operation)
-        for (shape in nestedShapes) {
-            when (shape.type) {
-                ShapeType.STRUCTURE -> renderEquatable(shape)
-                ShapeType.UNION -> renderEquatable(shape)
-                else -> {}
-            }
-        }
```

Comment:
Getting nested shapes for the test's response or error, and marking them so they get conformance added to them later, is now taken care of with a model transformation - see `EquatableConformanceTransformer` below.